### ISSUE TITLE
Publishing the code

### ISSUE THREAD

**Issue by zanilzanzan** [phiphiphi31/SuperSBT#1]
Dear authors, thank you for your inspiring work! The results look pretty promising, and looks like it could even outperform SwinTrack, which is the SOTA on several benchmarks at the moment. I am really excited to try the tracker out and see how it compares. Are you thinking of releasing the code any time soon? Best!

**Comment by phiphiphi31** [comment]
yeah, currently I donot have enough GPUs, so the remaining results need some time to be released. The implementation of my work SBT is very easy, you can try to reproduce it .  I recommend you to see the code of PVT (pyramid vision transformer) . If you are interested in this work. Please see the paper <correlation aware deep tracking>.

**Comment by phiphiphi31** [comment]
code, model, results are released.
